src/eval.jl

```diff
@@ -1,6 +1,7 @@
 using CodeTools, LNR, Media
 import CodeTools: getthing
 
+preprocess(x) = x
```

Comment:
I tried running
```
import Atom
Atom.preprocess(text) = string("import ChainRecursive\nChainRecursive.@chain begin\n", text, "\nend")
```
from within the Juno but it didn't change behavior immediately. Will this change post 265?